### PR TITLE
chore: release 0.2.0

### PR DESCRIPTION
**Pull request by haverchuck**
*Description of changes:*
chore: release 0.2.0 with support for null safety

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by offlineprogrammer — APPROVED** [review]

**Review by cshfang — APPROVED** [review]
Is our plan to alway pin to specific versions sans-caret from now on? Lgtm either way

File: packages/amplify_analytics_pinpoint/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+## 0.2.0 (2021-06-30)
+
+### Features
+
+- feat(amplify_analytics_pinpoint): Null safety (#483)
+- feat: Null safety core (#492)
+
+### Chores
+
+- chore: Null safety master rebase (#676)
```
Comment (Jordan-Nelson) [inline packages/amplify_analytics_pinpoint/CHANGELOG.md:10]: *nit:* I would probably omit this. I can't think of a scenario where this info would be particularly useful in a change log. I can see the counter argument of wanting to include it for completeness though.

**Review by Jordan-Nelson — APPROVED** [review]

File: packages/amplify_datastore/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
+## 0.2.0 (2021-06-30)
+
+### Features
```
Comment (cshfang) [inline packages/amplify_datastore/CHANGELOG.md:3]:
```suggestion
## 0.2.0 (2021-06-30)

### Breaking Changes

- The previously deprecated configureModelProvider function has been removed.

### Features
```

File: packages/amplify_analytics_pinpoint/CHANGELOG.md (hunk 1 above)
Comment (haverchuck) [reply packages/amplify_analytics_pinpoint/CHANGELOG.md:10]: Normally I'd agree, but this was such a massive change I'd err on leaving it in?

File: packages/amplify_datastore/CHANGELOG.md (hunk 2 above)
Comment (Ashish-Nanda) [reply packages/amplify_datastore/CHANGELOG.md:3]: We should include the breaking change that @cshfang mentioned.

**Review by Ashish-Nanda — APPROVED** [review]
LGTM 🚀

**Pull request merged**